packages/rich-text/src/helpers/editor.ts

```diff
@@ -286,10 +288,15 @@ export function currentSelectionPrecedesTableCell(editor: PlateEditor): boolean
   );
 }
 
+export const replaceNode = (editor: Editor, path: Location, replacement: Node | Node[]) => {
+  Transforms.removeNodes(editor, { at: path });
+  Transforms.insertNodes(editor, replacement, { at: path });
+};
+
 /**
  * It filters out all paragraphs and headings from a path and convert them into paragraphs.
  */
-export function slateNodeEntryToText(editor: PlateEditor, path: Path): CustomElement[] {
+export function getTextAt(editor: PlateEditor, path: Path): CustomElement[] {
```

Comment:
We need to be careful with the lists PR, it's currently using this helper as the previous name, so it will depend which one gets merged first.

Reply:
It will be a small conflict I think. Anyway, I'm not very happy with the name `getTextAt`, it's better than `slateNodeEntryToText` but doesn't quite describe what the function does

Reply:
What about `toParagraphs` ? It's `Paragraph`s, not `Text` nodes that we're returning so I'd opt for more accurate naming. No strong opinion on the rest of the naming.

Reply:
Renamed it to `extractParagraphsAt`